test/nodetool/test_cleanup.py

```diff
@@ -8,7 +8,7 @@
 import utils
 
 
-def test_cleanup(nodetool):
+def test_cleanup(nodetool, scylla_only):
```

Comment:
I think we should also cover what cassandra-nodetool does here.
We can either add a separate `cassandra_only` test, or add an if here, on which implementation is used. Can be a follow-up.